src/plugins/dam/components/uploader/FileList.jsx

```diff
@@ -35,19 +35,7 @@ class FileList extends React.Component {
   }
 
   componentDidUpdate() {
-    // Raven is making `componentDidUpdate` fire on each heart beat thus
-    // the uploader modal jumps when users with a small screen (like a 13" mbp)
-    // scrolls down. This will check if the user's browser is about a 13" screen
-    // or less and prevent the scrolling into view from happening.
-    if (document.documentElement.clientHeight > 792) {
-      this.ensureActiveItemVisible();
-    }
-  }
-
-  ensureActiveItemVisible() {
-    if (this.activeFileItem) {
-      scrollIntoViewIfNeeded(this.activeFileItem, { duration: 500 });
-    }
+    setTimeout(() => this.ensureActiveItemVisible(), 2000);
```

Comment:
@digitalartcrew this wont stop it from running on every update, it just delays it. use lodash throttle to make it so that it only fires at most once every 5 seconds https://lodash.com/docs/4.17.15#throttle